packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BiContractActivationWindow.jsx

```diff
@@ -108,40 +89,25 @@ const BIContractActivationWindow = ({
           variant="text"
           color="primary"
           disabled={!extraParams}
-          onClick={() => setWindowVisible(true)}
+          onClick={() => {
+            openOAuthWindow({
+              client,
+              konnector,
+              account,
+              extraParams,
+              intentsApi,
+              webviewIntent,
+              manage: true
+            })
+              .then(afterOAuthWindow)
```

Comment:
Why no await here? Is it because of a TS issue?

Reply:
No, since we want to call window.open as soon as possible, this removes one step compared to :

```javascript
async runAsync => {
  const result = await openOAuthWindow(...)
}
runAsync()
```